tests/integration/utils2.go

```diff
@@ -1204,11 +1199,12 @@ func createDocViaColSave(
 		return nil, err
 	}
 
-	return doc, collections[action.CollectionID].Save(
-		s.ctx,
-		acpIdentity.NewIdentity(action.Identity),
-		doc,
-	)
+	txn := getTransaction(s, node, immutable.None[int](), action.ExpectedError)
+
+	ctx := db.SetContextTxn(s.ctx, txn)
```

Comment:
question: Does setting order matter? 

suggestion: Even if order doesn't matter suggest, perhaps a function that sets all contexts so it's less error prone and consistent in all places we need to set both.

Reply:
The order does not matter. If we want to create a private function for the test utils and cli specifically that's fine with me. But a global function would be more error prone than the explicitness of setting each individually pretty much everywhere else we need them.